ui/src/features/SharedCanvas/index.tsx

```diff
@@ -58,6 +58,7 @@ const SharedCanvas: React.FC<Props> = ({
     <div className="relative flex size-full flex-col">
       <EditorProvider value={editorContext}>
         <Canvas
+          isSharedCanvas={true}
```

Comment:
If you make the change in Canvas,
```suggestion
          isSharedCanvas
```